test/infrastructure/clusters/test-infra/karpenter-tests/kustomization.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - pipeline-cleanup-cron.yaml
+  - pipeline-trigger-cron.yaml
+  - sweeper-cron.yaml
+  - rbac.yaml
+  - test-suites.yaml
+configMapGenerator:
+  - name: scripts
+    namespace: karpenter-tests
+    files:
+      - scripts/pipeline-cleanup.sh
+      - scripts/pipeline-trigger.sh
+      - scripts/sweep.sh
```

should this match the file name?
```suggestion
      - scripts/cleanup.sh
```

---

Good catch